core-bundle/contao/templates/_new/component/_headline.html.twig

```diff
@@ -21,7 +21,11 @@
 
     <{{ tag_name }}{% block headline_attributes %}{{ headline.attributes|default }}{% endblock %}>
     {%- block headline_inner -%}
-        {% if as_editor_view %}{{ headline.text }}{% else %}{{ headline.text|insert_tag_raw }}{% endif %}
+        {% if as_editor_view %}
+            {{ headline.text }}
+        {% else %}
+            {{ headline.text|insert_tag_raw }}
+        {% endif %}
```

Comment:
```suggestion
        {% if as_editor_view %}
            {{- headline.text -}}
        {% else %}
            {{- headline.text|insert_tag_raw -}}
        {% endif %}
```

Reply:
Changed in 8bd552f5952b80a01c9ef721233a31191f98e8f0.